src/AasxServerBlazor/Shared/MainLayout.razor

```diff
@@ -1,374 +1,207 @@
 @inject IRegistryInitializerService AasRegistryService
-@using System
 @using IO.Swagger.Registry.Lib.V3.Interfaces;
-@using Microsoft.IdentityModel.Tokens;
+@using System.Reflection
 
 <style>
-    .loader {
-    width: 28px;
-    height: 28px;
-    border: 5px solid #AAA;
-    border-bottom-color: transparent;
-    border-radius: 50%;
-    display: inline-block;
-    box-sizing: border-box;
-    animation: rotation 1s linear infinite;
+    @@keyframes rotation {
+        0% {
+            transform: rotate(0deg);
+        }
+        100% {
+            transform: rotate(360deg);
+        }
+    } 
+    
+    header {
+        position: fixed;
+        top: 0;
+        width: 100%;
+        box-shadow: 0 0.25rem 0.25rem rgba(0, 0, 0, 0.15);
+        z-index: 1000;
+    }
+    
+    .main {
+        flex: 1;
+        overflow-y: auto;
+        padding-top: 0.5rem;
+        padding-bottom: 4rem;
+    }
+    
+    .navbar {
+        display: flex;
+        justify-content: space-around;
+        padding: 10px 0;
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        background-color: white;
+        box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.1);
     }
 
-    @@keyframes rotation {
-    0% {
-        transform: rotate(0deg);
+    .logo {
+        max-height: 3.63rem;
+        margin-left: 0.3125rem;
+        margin-right: 0.3125rem;
     }
-    100% {
-        transform: rotate(360deg);
+
+    .logo-container {
+        display: flex;
+        align-items: center;
     }
-    } 
+
+    .nav-link {
+        margin-right: 20px;
+    }
+
+    .content {
+        padding: 20px 0;
+        margin-top: 5.5rem;
+        text-align: center;
+        z-index: 0;
+    }
+    
+    .loading {
+        color: white;
+        background-color: blue;
+    }
+    
+    .footer {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        background-color: #f8f9fa;
+        padding: 0.625rem 0;
+        text-align: center;
+        box-shadow: 0 -0.125rem 0.25rem rgba(0, 0, 0, 0.1);
+        z-index: 999;
+    }
+    
+    .footer div {
+        display: inline-block;
+        margin: 0 0.625rem;
+    }
+
 </style>
 
-@*<div class="sidebar">
-    <NavMenu />
-</div>*@
-<div class="main h-screen">
-    @{
-        if (Program.isLoading)
+<div class="main">
+    @if (Program.isLoading)
+    {
+        <span class="loading">Loading...</span>
+        System.Threading.Thread.Sleep(1000);
+        StateHasChanged();
+    }
+    else
+    {
+        InitializeCredentialsAndRegistry();
+
+        var isProductCarbonFootprintShowcase = IsProductCarbonFootprintShowcaseUri(NavigationManager.Uri);
+
+        if (isProductCarbonFootprintShowcase)
         {
-            <span style="color:white;background-color:blue;">Loading...</span>
-            System.Threading.Thread.Sleep(1000);
-            StateHasChanged();
+            <CarbonFootprintShowcaseLayout></CarbonFootprintShowcaseLayout>
```

Comment:
I moved all of the showcase stuff into the [CarbonFootprintShowcaseLayout.razor](https://github.com/eclipse-aaspe/aasx-server/pull/243/files#diff-19852ae1d8b2a6590b606f85d9d5114c77b7a60e6e6034d90629106e17dec3a8) to have it separated. I did not change any of the code there.